src/List/List.js

```diff
@@ -0,0 +1,3 @@
+import basicClassCreator from '../utils/basicClassCreator';
```

Comment:
You can move this in a `src/List/index.js` with the other imports as well. No need of a specific file since the end user will never import only this one (he won't be able to do much only with this)